packages/shared/src/components/CombatReport/index.tsx

```diff
@@ -85,6 +88,16 @@ export const CombatReportInternal = () => {
         >
           Replay
         </a>
+        {user?.me?.tags?.includes('rawlogs') && (
```

Comment:
is this supposed to be a public user feature or dev only? if dev only, we can also use env as the condition

Reply:
it's helpful to be able to turn it on for power users or to help troubleshoot remote issues